packages/xo-server-backup-reports/src/index.js

```diff
@@ -330,9 +573,9 @@ class BackupReportsXoPlugin {
                 operationLog.end - operationLog.start
               )}`
             )
-          } else if (get(operationLog.result, 'message') !== undefined) {
+          } else if (operationLog.result?.message !== undefined) {
```

Comment:
Why not using `getErrorMarkdown` everywhere?

Reply:
Because i don't modified the implementation of the backups report.

Reply:
Ok, while we're at it I think we should make our code uniform.